model/Core/Vocabularies/RelationshipType.md

```diff
@@ -25,28 +25,28 @@ name completes the sentence:
 
 ## Entries
 
-- affects: (Security/VEX) The `from` vulnerability affects each `to` Element. The use of the `affects` type is constrained to `VexAffectedVulnAssessmentRelationship` classed relationships.
```

Comment:
I don't remember the discussion of removing the profiles that are infront of relationships.   When did this happen?

Reply:
What do these (profile names in vocabulary entries) even mean? We never explain this.

Do we mean to say that you can **not** use `testedOn` unless you also comply with the AI/Dataset profile? (either? both?). Or it was simply introduced there?

Anyway these are not consistent, since for example "(Licensing)" does not appear.

Reply:
Agree that the information inside (...) is not explained anywhere inside RelationshipType summary/description.

If to leave the (...)s, may have to add a short line describes what they are, plus adding missing (...)s, like the mentioned "(Licensing)".

Reply:
OK,  I'll go along with making it consistent.   Probably something we need to go into more details with as a section in "Using" at any rate.

Reply:
This may related to "Translating from 2.3 to 3.0" table? https://github.com/spdx/spdx-spec/blob/development/v3.0.1/docs/annexes/diffs-from-previous-editions.md#translating-from-23-to-30-8

The table shows `LifecycleScopeType` expected to be used with a relationship type.

Reply:
Have reading through the relationship type list. I think it's quite safe to remove the information in (...)s.

For all `(Security)`/`(Security/VEX)` relationship types, their description have the word "Vulnerability" inside, which already give a context. Removing info in (...)s should be fine.

For `hasEvidence: (Dataset)` relationship type, we can say that the type can be use in a wider context than only Dataset. So removing `(Datatset)` is ok.

> - hasEvidence: (Dataset) Every `to` Element is considered as evidence for the `from` Element (`from` hasEvidence `to`).

For `(AI, Dataset)` relationship types, they can be a bit problematic if the context inside (...)s are to be removed.

> - testedOn: (AI, Dataset) The `from` Element has been tested on the `to` Element(s).
> - trainedOn: (AI, Dataset) The `from` Element has been trained on the `to` Element(s).

Maybe not so much for `trainedOn`.
But for `testedOn`, it can be confused with `hasTest` and `hasTestCase`.

If we're going to remove info in (...)s, I suggest we add few more context in the descriptions of `testedOn` (and maybe also `trainedOn`).